Extensions/xkit_patches.js

```diff
@@ -2136,6 +2137,10 @@ XKit.tools.getParameterByName = function(name){
 					m_return.search = true;
 				}
 
+				if (document.location.href.indexOf("tumblr.com/explore/") !== -1) {
```

Comment:
I would prefer to have a body class over not having a body class, if one exists on the page